data/quests.ts

```diff
@@ -1218,4 +1242,583 @@ export const Quests: Array<ListType> = [
+        id: "1aeeaf5b-347f-4adb-bb27-1b8a38cbce05",
+        description: "Progress Jar-Bairn's questline then exhaust both Diallos' and Jar-Bairn's dialogue, reloading the area as needed. Jar-Bairn should mention poachers at the end."
+      },
+      {
+        id: "b3143ed6-428c-4330-b975-ffe665c22b8a",
+        description: "After Jarburg is attacked (it may take waiting for time to pass), exhaust Diallos' and Jar-Bairn's dialogue again. When Diallos asks a question, you can respond with either option."
+      },
+      {
+        id: "6b6e87f3-8d39-4c96-bdae-7ae11d0fd6a2",
+        description: "Reload the area, exhaust Jar-Bairn's dialogue, then reload the area again to find Diallos's Mask, Hoslow's Petal Whip, and a Numen's Rune where Diallos was."
+      }
+    ]
+  },
+  {
+    id: "7283be28-9d16-40ba-b977-39d16df5d32b",
+    name: "Ensha",
+    url: "https://eldenring.wiki.fextralife.com/Ensha",
+    requirements: [
+      {
+        id: "f4865803-9b86-443a-9d7e-743bf900b45c",
+        description: "Interact with Ensha at Roundtable Hold to obtain the What Do You Want? Gesture."
+      },
+      {
+        id: "61227be7-78e6-4fe1-b09a-e30773697d96",
+        description: "Obtain either half of the Haligtree Secret Medallion (Right or Left)."
+      },
+      {
+        id: "03fa2971-d014-4546-8823-574b44456c1c",
+        description: "Return to Roundtable Hold and Ensha will invade."
+      },
+      {
+        id: "3a02f02b-7c36-4308-b142-621860f8fa05",
+        description: "Obtain Clinging Bone after defeating Ensha and the Royal Remains set from where Ensha was originally standing outside Gideon Ofnir's room."
+      },
+      {
+        id: "2d5da311-6b48-4e85-953a-3168a156f7ac",
+        description: "Talk to Gideon Ofnir and he will apologize for Ensha's behavior."
+      }
+    ]
+  },
+  {
+    id: "16d5085f-b29c-433d-ba3e-9edfcbd02856",
+    name: "Sorcerer Rogier",
+    url: "https://eldenring.wiki.fextralife.com/Sorcerer+Rogier",
+    requirements: [
+      {
+        id: "f9ff2515-7c1a-40ea-9f39-1bef1e83bb3e",
+        description: "Talk to Rogier in Stormveil Castle, by the altar of the Chapel, before defeating Godrick."
+      },
+      {
+        id: "a3a77ba3-1480-4df6-9446-8a467a900293",
+        description: "Talk to Rogier in Roundtable Hold and obtain Rogier's Rapier +8 with Ash of War: Glintblade Phalanx if you talked to him in Stormveil."
+      },
+      {
+        id: "b2cdf80e-a769-419d-a0c7-da59ded7b438",
+        description: "Defeat the Lesser Ulcerated Tree Spirit in Stormveil and view the memory in front of the giant face before climbing up the ladder."
+      },
+      {
+        id: "1790ace6-e10c-4055-bc5d-7b872c3e184a",
+        description: "Return to the Roundtable Hold and talk to Rogier."
+      },
+      {
+        id: "87e32e4e-50ee-42d6-a17a-9196800316a9",
+        description: "(Optional) Interact with Fia and select \"Hold Me\". Exhaust the extra dialogue options to receive Knifeprint Clue."
```

Comment:
I normally use template literals to use double quotes in this project. 
```suggestion
        description: `(Optional) Interact with Fia and select "Hold Me". Exhaust the extra dialogue options to receive Knifeprint Clue.`
```